src/lib.rs

```diff
@@ -864,3 +864,16 @@ mod tests {
         assert_eq!(settings.flow_control(), Some(FlowSoftware));
     }
 }
+
+
+/// A device-indepenent implementation of serial port information.
+#[derive(Debug,Clone,PartialEq,Eq)]
+pub struct SerialPortInfo {
```

Comment:
`PortInfo` seems sufficient to me, since `Serial` is already part of the crate name. Traits have `Serial` as part of their name to avoid name conflicts when imported through the prelude, but that's not an issue for structs.